chains/container-chains/runtime-templates/simple/src/xcm_config.rs

```diff
@@ -195,18 +197,39 @@ pub type XcmRouter = WithUniqueTopic<(
     SovereignPaidRemoteExporter<
         UmpRouter,
         UniversalLocation,
-        crate::EthereumNetwork,
+        EthereumNetwork,
         ContainerToEthTransferFee,
     >,
 )>;
 
+/// Filter to ensure an ETH asset is coming from a trusted Ethereum location.
```

Comment:
The same implementation as above would be great if we extract it to the separate file. 
Maybe let's keep it consistent with [this](https://github.com/moondance-labs/tanssi/pull/1276/files#r2358051760) PR. 
Put this file to `primitives/xcm-commons/src/lib.rs` and maybe use the same name, so we don't have issues later with conflicts.